test/karma.config.mjs

```diff
@@ -0,0 +1,167 @@
+import chalk from 'chalk';
+import { startGoldensServer } from './GoldensServer.mjs';
+
+// import { babel } from '@rollup/plugin-babel';
+// import json from '@rollup/plugin-json';
+// import ts from '@rollup/plugin-typescript';
+// import { nodeResolve } from '@rollup/plugin-node-resolve';
+// import commonjs from '@rollup/plugin-commonjs';
+//
+// const plugins = [
+//   json(),
+//   ts(),
+//   babel({
+//     extensions: ['.ts', '.js'],
+//     babelHelpers: 'bundled',
+//   }),
+//   nodeResolve(),
+//   commonjs({ include: 'node_modules/**' })
+// ];
```

Comment:
once we merge this I will open a follow up that bundles tests files so we can start using esm and jest

Reply:
Why is that needed? today isn't.